### ISSUE TITLE
Add tag or some other option to not always get annoyingly teleported to the lobby when in spectator mode, while you are trying to build an arena. ARRRRGH

### ISSUE THREAD

**Issue by NeunEinser** [slicedlime/Entrapment#21]
I always do /datapack disable, but to interact with the command blocks to save / load it, it has to be enabled and I always forget to disable it, it is really, really, *really* annoying.

**Comment by Kri5t0fK** [comment]
Local fix for yourself - put your nickname instead of slicedlime's here (line 29) - will prevent switching you from `spectator` to `adventure`:
https://github.com/slicedlime/Entrapment/blob/6c7e9950ade5cef3042e5771e56d564b244c2c66/datapacks/Entrapment/data/entrapment/functions/loop/lobby.mcfunction#L27-L29

**Comment by Kri5t0fK** [comment]
I fixed it by replacing restriction of having `name=slicedlime` to having `tag=DEV` 
in my [`chest-system`](https://github.com/Kri5t0fK/Entrapment/tree/chest-system) branch in 7dc7aa207f5961654733867214be060f5d107fa2
(just give yourself `/tag @s add DEV`)

**Comment by NeunEinser** [comment]
Hahaha I forgot I created this issue.

Yeah that sounds like an issue I would create when being mad at something :D

Are you planning to PR to the main repo? Maybe it'd make sense to cherry pick the commit for a seperate PR too

**Comment by Kri5t0fK** [comment]
Yes, I'm going to PR when I add `config` and `restrictions` I mentioned _at this time_ in README: [`Kri5t0fK/chest-system` at 31.08.21](https://github.com/Kri5t0fK/Entrapment/tree/03e566ba4c88017449113b7fe4b2286d63202594)
(assuming Lime accepts the chest-system ;P)
